bindings/c/test/apitester/run_c_api_tests.py

```diff
@@ -79,11 +82,15 @@ def run_tester(args, test_file):
     get_logger().info('')
     return ret_code
 
+def include_bg_test(bg_enabled, fname):
```

Comment:
Better introduce a separate directory for the BG tests

Reply:
That seems reasonble.
My thought here was that we still may want to run the non-bg tests on a cluster with blob granules configured and with a blob_worker process in case it affected anything or caught any issues, but that is less valuable.